src/VisualStudio/Core/Def/DocumentOutline/DocumentOutlineControl_ModelComputation.cs

```diff
@@ -154,33 +150,29 @@ private void EnqueueHighlightExpandAndPresentItemsTask(ExpansionOption expansion
         private async ValueTask HighlightExpandAndPresentItemsAsync(ImmutableSegmentedList<ExpansionOption> expansionOption, CancellationToken cancellationToken)
         {
             var model = await _filterAndSortDataModelQueue.WaitUntilCurrentBatchCompletesAsync().ConfigureAwait(false);
-            if (model is null)
-                return;
 
-            // Switch to the UI thread to get the current caret point and latest active text view then create the UI model.
+            // Switch to the UI thread to get the current caret point and create the UI model.
             await _threadingContext.JoinableTaskFactory.SwitchToMainThreadAsync(cancellationToken);
 
             var activeTextView = GetLastActiveIWpfTextView();
-            if (activeTextView is null)
-                return;
-
-            var caretPoint = activeTextView.GetCaretPoint(activeTextView.TextBuffer);
-            if (!caretPoint.HasValue)
-                return;
+            var caretPoint = activeTextView?.GetCaretPoint(activeTextView.TextBuffer);
 
-            var documentSymbolUIItems = DocumentOutlineHelper.GetDocumentSymbolUIItems(model.DocumentSymbolData, _threadingContext);
+            var documentSymbolUIItems = model is null ?
+                ImmutableArray<DocumentSymbolUIItem>.Empty :
+                DocumentOutlineHelper.GetDocumentSymbolUIItems(model.DocumentSymbolData, _threadingContext);
```

Comment:
Nit:  THis makes it nice and easy to see that it's a conditional and what the different branches of it are.

```suggestion
            var documentSymbolUIItems = model is null
                ? ImmutableArray<DocumentSymbolUIItem>.Empty
                : DocumentOutlineHelper.GetDocumentSymbolUIItems(model.DocumentSymbolData, _threadingContext);
```